cfme/cloud/flavor.py

```diff
@@ -210,6 +211,16 @@ def create(self, name, provider, ram, vcpus, disk, swap, rxtx, is_public=True, c
         flavor = self.instantiate(name, provider, ram, vcpus, disk, swap, rxtx, is_public)
         return flavor
 
+    def all(self):
+        provider = self.filters.get('provider')  # None if no filter, need for entity instantiation
+        view = navigate_to(self, 'All')
+        result = []
+        for e in view.entities.get_all():
```

Comment:
Required:
It will not cover all entities. I think we should go with `surf_pages=True`. It will cover all `entities` even if more pages.  same suggestion of other as well.